docs/core_docs/docs/integrations/platforms/microsoft.mdx

```diff
@@ -100,20 +100,36 @@ See a [usage example](/docs/integrations/vectorstores/azure_aisearch).
 import { AzureAISearchVectorStore } from "@langchain/community/vectorstores/azure_aisearch";
 ```
 
+### Azure Cosmos DB for NoSQL
+
+> [Azure Cosmos DB for NoSQL](https://learn.microsoft.com/azure/cosmos-db/nosql/) provides support for querying items with flexible schemas and native support for JSON. It now offers vector indexing and search. This feature is designed to handle high-dimensional vectors, enabling efficient and accurate vector search at any scale. You can now store vectors directly in the documents alongside your data. Each document in your database can contain not only traditional schema-free data, but also high-dimensional vectors as other properties of the documents. 
+
+<IntegrationInstallTooltip></IntegrationInstallTooltip>
+
+```bash npm2yarn
+npm install @langchain/azure-cosmosdb
```

Comment:
Depending on how many integrations you're planning - do you think it makes sense to have a single `@langchain/azure` package with peer dependencies on different packages?

Reply:
If you're ok with that, I think smaller packages might be more manageable and easier to maintain, as opposed to one big package. I don't think we'll create too many packages, though I don't have the plans of all product teams 🙂